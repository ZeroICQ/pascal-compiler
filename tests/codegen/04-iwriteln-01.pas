 begin
    iwriteln(20);
    iwriteln(220);
    iwriteln(240);
 end.