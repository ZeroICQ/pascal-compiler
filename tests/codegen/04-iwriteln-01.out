20
220
240